core/azure-core/src/main/java/com/azure/core/http/policy/spi/BeforeRetryPolicyProvider.java

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.core.http.policy.spi;
+
+/**
+ * Implementing classes are automatically added as policies before the retry policy.
+ */
```

Comment:
I thought I had this under the implementation package when I did it? Did you move it? I don't think the classes in the /spi package need to be public API quite yet - thoughts?

Reply:
Moved to implementation